nautobot/users/filters.py

```diff
@@ -59,27 +94,22 @@ class TokenFilterSet(BaseFilterSet):
 
     class Meta:
         model = Token
-        fields = ["id", "key", "write_enabled", "created", "expires"]
+        fields = ["id", "key", "write_enabled", "created", "expires", "description"]
 
 
 class ObjectPermissionFilterSet(BaseFilterSet):
-    user_id = django_filters.ModelMultipleChoiceFilter(
+    users = NaturalKeyOrPKMultipleChoiceFilter(
         field_name="users",
```

Comment:
This line might be redundant now?

Reply:
I don't think so, 🤔?

Reply:
since the filter name is now `users` I thought the `field_name="users"` was not needed since that will be the default. I may be mistaken.

Reply:
oooh okay